### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #172: Code of Conduct, Contributing guidelines etc.
Hi @paulcsmith.
Is there anything planned regarding a [CoC](https://thoughtbot.com/open-source-code-of-conduct) or contributing guidelines (like https://github.com/thoughtbot/factory_bot/blob/master/CONTRIBUTING.md)?

How could I help with this? Would you rather use existing documents from other Thoughtbot projects, or would you use different ones?

### PR DESCRIPTION
**Pull request by 5minpause**
Closes #172

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Contributing to Lucky Framework
```
Comment (paulcsmith) [inline CONTRIBUTING.md]: Could you change this to just "Lucky"?

**Comment by paulcsmith** [comment]
Actually I can just change that one line myself pretty quickly. Thanks again for doing this 👍

**Comment by 5minpause** [comment]
I am just happy to contribute to this framework. I think it's exciting.

**Comment by paulcsmith** [comment]
@5minpause I'm happy to hear that! BTW, if you have a moment, could you comment here with the things that you've liked so far, and also some things that you aren't sure about or that could use improvement? I'd love to hear your feedback

**Pull request merged**